Resources/Prototypes/Nyanotrasen/Roles/Jobs/Engineering/cyborg.yml

```diff
@@ -4,8 +4,11 @@
   description: job-description-cyborg
   playTimeTracker: JobCyborg
   requirements:
+    - !type:DepartmentTimeRequirement
+      department: Engineering
+      time: 14400
```

Comment:
I don't generally consider this to be an Engineering Borg, and I don't think many others do either. The existence of their hypospray I feel removes any strong correlation to Engi. They kinda just float around doin stuff.